moto/route53/urls.py

```diff
@@ -36,8 +30,10 @@ def tag_response2(request: Any, full_url: str, headers: Any) -> TYPE_RESPONSE:
     r"{0}/(?P<api_version>[\d_-]+)/healthcheck$": Route53.dispatch,
     r"{0}/(?P<api_version>[\d_-]+)/healthcheck/(?P<health_check_id>[^/]+)$": Route53.dispatch,
     r"{0}/(?P<api_version>[\d_-]+)/healthcheck/(?P<health_check_id>[^/]+)/status$": Route53.dispatch,
-    r"{0}/(?P<api_version>[\d_-]+)/tags/healthcheck/(?P<zone_id>[^/]+)$": tag_response1,
-    r"{0}/(?P<api_version>[\d_-]+)/tags/hostedzone/(?P<zone_id>[^/]+)$": tag_response2,
+    r"{0}/(?P<api_version>[\d_-]+)/tags/healthcheck/(?P<zone_id>[^/]+)$": tag_response,
+    r"{0}/(?P<api_version>[\d_-]+)/tags/hostedzone/(?P<zone_id>[^/]+)$": tag_response,
```

Comment:
```suggestion
    r"{0}/(?P<api_version>[\d_-]+)/tags/hostedzone/(?P<zone_id>.+)$": tag_response,
```
I believe this is the reason the test is currently failing in ServerMode. The `zone_id` contains url-encoded slashes (`%2Fhostedzone%2Fzone_id`), but I guess the matcher does some un-encoding before trying to match the incoming URL - so this regex needs to accept a `zone_id` that can contain slashes.

Reply:
Gotcha, thanks @bblommers for catching this and making the change! One question though – wouldn't this also then apply to the `/tags/healthcheck` url as well?